docs/components/Typography/TextStylesPage.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { ComponentStory } from '@storybook/react';
+
+import TextStylesPage from './TextStylesPage';
+export default {
+  title: 'Docs/Components/Typography',
```

Comment:
Ian updated the titles to have a different folder organization, you may want to update this title to  match what he has, I think it would be `Typography/Examples`

Reply:
Have been changed!